Parsers/ASimDns/Parsers/ASimDnsFortinetFortigate.yaml

```diff
@@ -0,0 +1,213 @@
+          "19", "BADMODE", "Failure",
+          "20", "BADNAME", "Failure",
+          "21", "BADALG", "Failure",
+          "22", "BADTRUNC", "Failure",
+          "23", "BADCOOKIE", "Failure"
+      ];
+      let DnsQueryTypeLookup = datatable(DnsQueryType:int, DnsQueryTypeName:string)[
+          0, "Reserved",
+          1, "A",
+          2, "NS",
+          3, "MD",
+          4, "MF",
+          5, "CNAME",
+          6, "SOA",
+          7, "MB",
+          8, "MG",
+          9, "MR",
+          10, "NULL",
+          11, "WKS",
+          12, "PTR",
+          13, "HINFO",
+          14, "MINFO",
+          15, "MX",
+          16, "TXT",
+          17, "RP",
+          18, "AFSDB",
+          19, "X25",
+          20, "ISDN",
+          21, "RT",
+          22, "NSAP",
+          23, "NSAP-PTR",
+          24, "SIG",
+          25, "KEY",
+          26, "PX",
+          27, "GPOS",
+          28, "AAAA",
+          29, "LOC",
+          30, "NXT",
+          31, "EID",
+          32, "NIMLOC",
+          33, "SRV",
+          34, "ATMA",
+          35, "NAPTR",
+          36, "KX",
+          37, "CERT",
+          38, "A6",
+          39, "DNAME",
+          40, "SINK",
+          41, "OPT",
+          42, "APL",
+          43, "DS",
+          44, "SSHFP",
+          45, "IPSECKEY",
+          46, "RRSIG",
+          47, "NSEC",
+          48, "DNSKEY",
+          49, "DHCID",
+          50, "NSEC3",
+          51, "NSEC3PARAM",
+          52, "TLSA",
+          53, "SMIMEA",
+          55, "HIP",
+          56, "NINFO",
+          57, "RKEY",
+          58, "TALINK",
+          59, "CDS",
+          60, "CDNSKEY",
+          61, "OPENPGPKEY",
+          62, "CSYNC",
+          63, "ZONEMD",
+          64, "SVCB",
+          65, "HTTPS",
+          99, "SPF",
+          100, "UINFO",
+          101, "UID",
+          102, "GID",
+          103, "UNSPEC",
+          104, "NID",
+          105, "L32",
+          106, "L64",
+          107, "LP",
+          108, "EUI48",
+          109, "EUI64",
+          249, "TKEY",
+          250, "TSIG",
+          251, "IXFR",
+          252, "AXFR",
+          253, "MAILB",
+          254, "MAILA",
+          255, "*",
+          256, "URI",
+          257, "CAA",
+          258, "AVC",
+          259, "DOA",
+          32768, "TA",
+          32769, "DLV"
+      ];
+      CommonSecurityLog
+      | where not(disabled)
+      | where DeviceVendor    == "Fortinet" and 
+          DeviceProduct       == "Fortigate"
+      | where DeviceEventClassID in(54000,54200,54400,54401,54600,54601,54800,54801,54802,54803,54804,54805)
+      | project TimeGenerated, EventOriginalSubType = DeviceEventClassID, AdditionalExtensions, EventUid = _ItemId, EventOriginalSeverity = LogSeverity, EventProductVersion = DeviceVersion ,Computer, Type, SrcIpAddr = SourceIP, SrcPortNumber = SourcePort, DstIpAddr = DestinationIP, DstPortNumber = DestinationPort, EventMessage = Message, NetworkProtocolNumber = Protocol
```

Comment:
I found below
https://docs.fortinet.com/document/fortigate/7.4.1/fortios-log-message-reference/671442/cef-priority-levels

Reply:
In Network we used :

  let SeverityLookup = datatable (EventOriginalSeverity:string, EventSeverity:string)
  [
    "1", "Informational", // Debug
    "2", "Informational", // Information
    "3", "Informational", // Notification
    "4", "Low", // Warning
    "5", "Low", // Error
    "6", "Critical", // High
    "7", "Alert", // Medium
    "8", "High" // Emergency
  ];

  But since we also do a lookup to DeviceEventClassIDLookup for the EventSubType and DvcAction it made more sense to add it to the table.